tensorflow/contrib/go/tensor_test.go

```diff
@@ -0,0 +1,630 @@
+package tensorflow_test
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	tf "github.com/tensorflow/tensorflow/tensorflow/contrib/go"
+)
+
+func getTensorFromGraph(t *testing.T, graphStr string) *tf.Tensor {
+	graph, err := tf.NewGraphFromText(graphStr)
+	if err != nil {
+		t.Fatal(err)
+	}
+	s, err := tf.NewSession()
+	if err := s.ExtendGraph(graph); err != nil {
+		t.Fatal(err)
+	}
+
+	output, err := s.Run(nil, []string{"output"}, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if len(output) != 1 {
+		t.Fatalf("The expexted number of tensors is 1 but there was %d tensors returned", len(output))
```

Comment:
typo 'expexted'
suggest:  t.Fatal("Expected one tensor but got:", len(output))
(or 1, whichever is consistent with what you choose in the other functions)